frontend/src/shared/components/Layout.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+The book project lets a user keep track of different books they would like to read, are currently
+reading, have read or did not finish.
+Copyright (C) 2020  Karan Kumar
+
+This program is free software: you can redistribute it and/or modify it under the terms of the
+GNU General Public License as published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program.
+If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import React, { Component } from 'react'
+import { NavBar } from '../navigation/NavBar'
+import './Layout.css'
+
+export function Layout(props: LayoutProps) {
+    const format = (props.centered ? "centered" : "");
+
+    return (
+        <div className="layoutContainer">
```

Comment:
It looks like kebab-case was used for all case styles in Goal.tsx/Goal.css, MyBooks.tsx/MyBooks.tsx etc.
Can we change classNames in `Layout.css` and `Layout.tsx` to match the other case styles? e.g.(.layout-container, .page-content, .page-title etc.)

Reply:
That's a good point! Do you know if kebab-case is the "right" style for style class names? In our [docs site](https://project-books.github.io/docs/styleguides/typescript-styleguide/) it looks like most names should be camelCase, but style class names aren't mentioned. @knjk04 do you have a preference?

Reply:
I also saw that the docs don't mention a preference for it and asked Karan if we should add that in to our docs :)

Reply:
We'll go back in a separate issue and adhere to the new style guide. I wouldn't want us to increase the blast radius of this PR